core/src/main/scala/org/ensime/core/FqnToSymbol.scala

```diff
@@ -50,9 +50,12 @@ trait FqnToSymbol { self: Global with SymbolToFqn =>
       segToSym(nme.segments(scalaName, assumeTerm = assumeTerm), rootSymbol)
     }
   }.getOrElse {
-    val term = segToSym(nme.segments(scalaName, assumeTerm = true), rootSymbol)
```

Comment:
I don't think these changes are doing anything... in a medium/large codebase it's best not to change code for stylistic reasons because a) your name will show up in the git history b) it mixes unrelated changes with your feature c) it can cause unexpected regressions that are super hard to track down.